types/chai-jquery/chai-jquery-tests.ts

```diff
@@ -24,7 +24,7 @@ function test_css() {
 function test_data() {
     expect($('#foo')).to.have.data('toggle');
     expect($('#foo')).to.have.css('toggle', 'true');
-    expect($('body')).to.have.css('font-family').and.match(/sans-serif/);
+    expect($('body')).to.have.css('font-family').and.match('sans-serif');
```

Comment:
Similarly to above question. Assertion in chai-dom using /sans-serif/ in their example. Could you explain why this change to string?

http://chaijs.com/plugins/chai-jquery/